packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,232 @@
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        name character varying(100) NOT NULL,
+        "applicationId" uuid
+    );
+    
+    ALTER TABLE public.environment OWNER TO postgres;
+    
+    CREATE TABLE public.portfolio (
+        id uuid DEFAULT public.uuid_generate_v4() NOT NULL,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "provisioningStatus" public.portfolio_provisioningstatus_enum DEFAULT 'not_started'::public.portfolio_provisioningstatus_enum NOT NULL,
+        administrators character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        name character varying(100) NOT NULL,
+        description character varying(300),
+        owner character varying NOT NULL,
+        csp public.portfolio_csp_enum NOT NULL,
+        "dodComponents" character varying[] NOT NULL,
+        "portfolioManagers" character varying[] NOT NULL
+    );
+    
+    ALTER TABLE public.portfolio OWNER TO postgres;
+    
+    CREATE TABLE public.task_order (
+        id uuid DEFAULT public.uuid_generate_v4() NOT NULL,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "taskOrderNumber" character varying(17) NOT NULL,
+        "fileId" uuid NOT NULL,
+        "fileName" character varying(256) NOT NULL,
+        "fileSize" integer,
+        "fileScanStatus" public.task_order_filescanstatus_enum DEFAULT 'pending'::public.task_order_filescanstatus_enum NOT NULL,
+        "portfolioId" uuid
+    );
+    
+    ALTER TABLE public.task_order OWNER TO postgres;
+    
+    COMMENT ON COLUMN public.task_order."taskOrderNumber" IS 'TO numbers are 13 characters. TO modifications are 17 characters.';
+    COMMENT ON COLUMN public.task_order."fileId" IS 'S3 object key of task order pdf';
+    COMMENT ON COLUMN public.task_order."fileName" IS 'name of pdf file when uploaded';
+    COMMENT ON COLUMN public.task_order."fileSize" IS 'pdf file size in bytes';
+    
+    
+    CREATE TABLE public.typeorm_metadata (
+        type character varying NOT NULL,
+        database character varying,
+        schema character varying,
+        "table" character varying,
+        name character varying,
+        value text
+    );
+    
+    
+    ALTER TABLE public.typeorm_metadata OWNER TO postgres;
+    
+    ALTER TABLE ONLY public.task_order
+        ADD CONSTRAINT "PK_15e973a09676013165565bc6243" PRIMARY KEY (id);
+    
```

Comment:
We also should decide, stylistically, whether we want to add constraints as ALTERs after the fact or as part of the `CREATE TABLE`. https://www.postgresql.org/docs/13/sql-createtable.html

Reply:
I like how all the constraints are in one place, but at the same time being added with the CREATE TABLE will probably reduce the code and keep the contraints closer to the table.

Reply:
Will include constraints in the CREATE TABLE statements while giving them more meaningful names as @jeffsegal-ccpo has suggested.

Reply:
Moved PK constraints in 5828e63801f4228ab9b073cca7803543aae51d10

Reply:
Moved FK constraints in a98c2c9d35fe3835f46e4e6faa67c81edd4c935b